VirtualReality/Logic/vtkSlicerVirtualRealityLogic.cxx

```diff
@@ -208,3 +208,36 @@ void vtkSlicerVirtualRealityLogic::SetVirtualRealityActive(bool activate)
     }
   }
 }
+
+//---------------------------------------------------------------------------
+void vtkSlicerVirtualRealityLogic::setDefaultReferenceView()
```

Comment:
Can you make it upper case please: SetDefaultReferenceView ?

Reply:
Good catch (forgot to update it when copied from Qt class)